README.md

```diff
@@ -30,13 +30,42 @@ This blueprint assumes that you are familiar with GKE and
 To deploy this blueprint you need:
 
 - A [Google Cloud project](https://cloud.google.com/docs/overview#projects) with billing enabled.
-- An account with the [Project Owner role](https://cloud.google.com/iam/docs/understanding-roles#resource-manager-roles) on the project.
```

Comment:
Instead of removing this line, refactor it by mentioning the two options, linking to the relevant sections below.

Reply:
I added